backend/app/tests/ui/pages/home.py

```diff
@@ -90,6 +94,14 @@ def click_create_button(self):
             self.wait.until(lambda _: self.page.create_job_modal_is_open)
             sleep(2)
 
+        @property
+        def distribution_preview_radio_button(self):
+            return self.find_element(*self._modal_preview_radio_button_locator)
+
+        def click_distribution_preview_radio_button(self):
+            self.distribution_preview_radio_button.click()
+            # self.wait.until(lambda _: self.page.create_job_modal_is_open)
```

Comment:
is this comment still needed?

Reply:
@tomjw64 aaahhhh... I just didn't see it there. I will delete it and re-push the changes for review

Reply:
@tomjw64 done. removed the comment.

Reply:
my approving review stands, you should be able to merge